lib/pages/route_creator.dart

```diff
@@ -0,0 +1,136 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:latlong2/latlong.dart';
+
+import '../widgets/app_header.dart';
+import '../widgets/complex_marker.dart';
```

Comment:
I'm not a big fan of relative paths and I would consider using package paths (hope that's the right name 😅)

Reply:
I'll fix both in next pr. Right now I merge